src/schema/schema_create.c

```diff
@@ -1133,6 +1133,7 @@ __create_layered(WT_SESSION_IMPL *session, const char *uri, bool exclusive, cons
      */
     WT_ERR(__wt_buf_fmt(session, tmp, "ingest=\"%s\",stable=\"%s\"", ingest_uri, stable_uri));
     layered_cfg[3] = tmp->data;
+    layered_cfg[4] = "log=(enabled=false)";
```

Comment:
I don't think this is right -- `layered_cfg` is only 5 elements long so the stuff at index 4 must be NULL so the config parser knows that's the end.